packages/resilience/src/resilience.spec.ts

```diff
@@ -0,0 +1,63 @@
+import axios from 'axios';
+import nock from 'nock';
+import { executeWithMiddleware } from './middleware';
+import { retry } from './retry';
+import { timeout } from './timeout';
+
+describe('combined resilience features', () => {
```

Comment:
[q] I like the separate file for the combined features. Could you have a look and move a combinded test from timeout.spec.ts if there is any.